From e5315dee63070c2ebfca9f5a63476cc9beb1c935 Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Thu, 30 May 2013 16:27:23 +0100
Subject: [PATCH 28/31] RHEL 6: tests/regressions/rhbz895904.sh: Use `` instead of $().

Apparently bash in RHEL 6 cannot handle this unusual
multi-line use of $(...), but `` works fine.
---
 tests/regressions/rhbz895904.sh |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/regressions/rhbz895904.sh b/tests/regressions/rhbz895904.sh
index 6440620..394f37a 100755
--- a/tests/regressions/rhbz895904.sh
+++ b/tests/regressions/rhbz895904.sh
@@ -23,7 +23,7 @@
 set -e
 export LANG=C
 
-output=$(
+output=`
 ../../fish/guestfish -N fs -m /dev/sda1 <<EOF | sort -k 3
 mkdir /test
 touch /test/file1
@@ -31,7 +31,7 @@ mkdir /test/subdir
 write /test/subdir/file2 abc
 checksums-out crc /test -
 EOF
-)
+`
 
 if [ "$output" != "4294967295 0 ./file1
 1219131554 3 ./subdir/file2" ]; then
-- 
1.7.1

